t/BikeSharingCitiBikeNYC.t

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+ddg_spice_test(
+    [qw( DDG::Spice::BikeSharing::CitiBikeNYC )],
+    'citi bike ny' => test_spice(
+        '/js/spice/bike_sharing/citi_bike_nyc/ny',
+        call_type => 'include',
+        caller => 'DDG::Spice::BikeSharing::CitiBikeNYC',
+    is_cached => 1,
+    ),
```

Comment:
The API doesn't seem to have any search feature. The best you can do is probably focus the searched area on the map. Make a list of approximate center coordinated for those 3 or 4 areas and focus them initially. Not sure if that's possible with the maps api. Haven't used it yet.